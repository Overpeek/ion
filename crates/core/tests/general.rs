#[test]
fn simple_general() {}
