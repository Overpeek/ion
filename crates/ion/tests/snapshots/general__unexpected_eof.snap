---
source: crates/core/tests/general.rs
expression: "err.pretty_print(false, code, \"<code>\")"
---
error: Unexpected EOF, expected one of "(", "false", "fn(", "true", r#"\"[^\"]*\""#, r#"[a-zA-Z_][\\w_]*"#, r#"\\.\\d[\\d_]*"#, r#"\\d[\\d_]*"#, r#"\\d[\\d_]*\\."# or r#"\\d[\\d_]*\\.\\d[\\d_]*"# --> <code>:1:4
  |
1 | x =
  |    ^

